### DISCUSSION TITLE
Is there a way to turn off the pro label without downgrading my plan?

### DISCUSSION THREAD

**Discussion by missmatsuko** [community/community#22347] [New to GitHub]
<p>I don’t really want people to know that I’ve paid for pro and possibly have hidden repos. It also just seems kind of elitist/cocky to me to display it like that.</p>
<p>I’ve already downgraded my plan, but I guess the label won’t go away until the current billing period is over?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/missmatsuko">@missmatsuko</a>,</p>
<p>Thank you for being here and for your feedback! You can now disable the Pro badge on your account by following the steps found here:</p>
<p><a href="https://help.github.com/articles/personalizing-your-profile/#disabling-the-github-pro-badge-on-your-profile" rel="nofollow noopener">https://help.github.com/articles/personalizing-your-profile/#disabling-the-github-pro-badge-on-your-profile</a></p>
<p>I hope this helps!</p>
<p>Andrea</p>

**Comment by missmatsuko** [comment]
<p>Thanks, I couldn’t find that info because I didn’t know it’s called a “badge”. “Badge” makes me think of the ones that are in repo READMEs.</p>
